### PR TITLE
Patches to get surfman to work

### PR DESCRIPTION
**Pull request by asajeffrey**
These are the patches to osmesa to get surfman to work. I filed issues for them upstream.

### CODE REVIEW

File: patches/mesa-issue-2034.patch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--- a/mesa-src/src/gallium/state_trackers/osmesa/osmesa.c
```
Comment (jdm) [inline patches/mesa-issue-2034.patch]: Please add these files to https://github.com/servo/osmesa-src/blob/6ba41c7d227be8cb745a176b53ae39fbeec99f0a/tools/update-mesa.sh#L35.

**Review by jdm — CHANGES_REQUESTED** [review]

File: patches/mesa-issue-2034.patch (hunk 1 above)
Comment (asajeffrey) [reply patches/mesa-issue-2034.patch]: Done.

**Pull request merged**